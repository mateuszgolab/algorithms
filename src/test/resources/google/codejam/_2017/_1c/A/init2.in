1
4 3
25802 2020
29226 2015
25389 2987
292154 42
