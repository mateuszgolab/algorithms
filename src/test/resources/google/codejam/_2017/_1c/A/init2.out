Case #1: 43982.297150257
Case #2: 625.176938064
